appinfo/info.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0"?>
-<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
-	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
+<info xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>end_to_end_encryption</id>
 	<name>End-to-End Encryption</name>
 	<summary>End-to-end encryption endpoint</summary>
 	<description>Provides the necessary endpoint to enable end-to-end encryption.</description>
-	<version>1.12.0-beta.1</version>
+	<version>1.12.1</version>
 	<licence>agpl</licence>
 	<author>Bjoern Schiessle</author>
 	<namespace>EndToEndEncryption</namespace>
 	<types>
-		<filesystem/>
-		<dav/>
+		<filesystem />
+		<dav />
 	</types>
 	<category>files</category>
 	<website>https://github.com/nextcloud/end_to_end_encryption</website>
 	<bugs>https://github.com/nextcloud/end_to_end_encryption/issues</bugs>
 	<repository type="git">https://github.com/nextcloud/end_to_end_encryption.git</repository>
-	<screenshot>https://raw.githubusercontent.com/nextcloud/end_to_end_encryption/master/screenshots/e2ee-filelisting.png</screenshot>
+	<screenshot>
+		https://raw.githubusercontent.com/nextcloud/end_to_end_encryption/master/screenshots/e2ee-filelisting.png</screenshot>
```

Comment:
```suggestion
	<screenshot>https://raw.githubusercontent.com/nextcloud/end_to_end_encryption/master/screenshots/e2ee-filelisting.png</screenshot>
```

Would like to avoid the risk of the appstore or something else breaking due to the line break

Reply:
Ah, stupid auto formatting. Fixed.